Lib/logging/__init__.py

```diff
@@ -1120,6 +1120,8 @@ def setStream(self, stream):
     def __repr__(self):
         level = getLevelName(self.level)
         name = getattr(self.stream, 'name', '')
+        #  bpo-36015: name can be an int
+        name = str(name)
```

Comment:
Just a thought: This could be shortened to:

```
#  bpo-36015: name can be an int
name = str(getattr(self.stream, 'name', ''))
```

However, doing it the way you've done it here makes it easier to indicate that the comment specifically applies to the `str(...)` part, so I'm torn on whether you should change it.

Reply:
For the reason stated, I think it's fine to have the `str()` call as a separate step.